po/fr.po

```diff
@@ -3,51 +3,48 @@
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 # Sergio Costas <rastersoft@gmail.com>, 2017.
+# Maïeul Rouquette <maieul@maieul.net>, 2018.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-04-23 22:50+0200\n"
-"PO-Revision-Date: 2017-09-14 00:12+0200\n"
-"Last-Translator: Sergio Costas <rastersoft@gmail.com>\n"
-"Language-Team: Español; Castellano <rastersoft@gmail.com>\n"
+"PO-Revision-Date: 2018-05-04 22:40+0200\n"
+"Last-Translator: Maïeul Rouquette <maieul@maieul.net>\n"
+"Language-Team: français <>\n"
 "Language: fr_FR\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Plural-Forms: nplurals=4; plural= (n==1) ? 0 : (n==2) ? 1 : (n != 8 && n != "
+"11) ? 2 : 3;\n"
 "X-Generator: Gtranslator 2.91.7\n"
 
 #. TRANSLATOR this message is shown in the configuration window to specify that cronopete is in idle state, not doing a backup
 #: src/backup_base.vala:99
 msgid "Ready"
-msgstr ""
+msgstr "Prêt"
 
 #. TRANSLATORS This is the name for the backend that allows to store the backups in a folder, instead of choosing a disk.
 #: src/backup_folder.vala:68
 msgid "Store backups in a folder"
-msgstr ""
+msgstr "Sauvegarder dans un dossier"
```

Comment:
"Sauvegarder les sauvegardes dans un dossier" is better.

Reply:
"Sauvegarder les sauvegardes" is pleonastic. Mabyer "conserver les sauvegardes".